src/lib/users.js

```diff
@@ -26,6 +28,28 @@ export async function getUserBySlug(slug) {
   };
 }
 
+/**
+ * authorPathByNameSlug
+ */
+
+export function authorPathByNameSlug(name) {
```

Comment:
not a big deal - but would we want this to be called `authorPathByName` without the `slug` on there since it would be the full name unformatted like `Colby Fayock`? it works either way but just a thought, as opposed to be low, where `getUserByNameSlug` would use `colby-fayock`

Reply:
Totally right! Updated